src/main/resources/liquibase/202302151544-remove-uuid-cast-on-instance-id-in-tally-instance-view.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+  <changeSet id="202302151544-1" author="mstead">
+    <dropView viewName="tally_instance_view"/>
+  </changeSet>
+
+  <changeSet id="202302151544-2" author="mstead">
+    <comment>Create view of hosts and host_tally_buckets for use in Instance API</comment>
+    <createView
+      replaceIfExists="true"
+      viewName="tally_instance_view">
+      select distinct org_id,
+                      h.id,
+                      h.instance_id as instance_id,
+                      display_name,
+                      h.billing_provider as host_billing_provider,
+                      h.billing_account_id as host_billing_account_id,
+                      b.billing_provider as bucket_billing_provider,
+                      b.billing_account_id as bucket_billing_account_id,
+                      last_seen,
+                      coalesce(num_of_guests, 0) AS num_of_guests,
+                      product_id,
+                      sla,
+        usage,
+        COALESCE(b.measurement_type, 'EMPTY') AS measurement_type,
+        m.uom,
+        m.value,
+        b.sockets,
+        b.cores,
+        h.subscription_manager_id
+      from hosts h
+        join host_tally_buckets b on h.id = b.host_id left outer join instance_measurements m on h.id = m.instance_id;
```

Comment:
Can you tidy this up?  It's kind of hard to read like this.  The indentation is inconsistent, the case is inconsistent (`coalesce` vs `coalesce`), some columns have the table prefix `h.` while others don't, and some columns are aliased when they don't need to be (`h.instance id as instance_id`).  I think you could just drop this text into DataGrip and run the formatter and it would fix a lot of the problems.

Reply:
DONE